BUILDING.md

```diff
@@ -273,39 +215,39 @@ Requirements:
 
 Get the following mxe packages:
 
-    $ make gcc sqlite qt nsis
+    make gcc sqlite qt nsis
 
 After successful compilation go into your mxedir/usr/bin and add 3 symlinks:
 
-    $ ln -s i686-pc-mingw32-windres windres
-    $ ln -s i686-pc-mingw32-makensis makensis
-    $ ln -s /usr/bin/lrelease
+    ln -s i686-pc-mingw32-windres windres
+    ln -s i686-pc-mingw32-makensis makensis
+    ln -s /usr/bin/lrelease
 
 Now cd into your sqlitebrowser source directory and create a build directory
 for the windows binary and create the correct makefiles:
 
-    $ mkdir build-win
-    $ cd build-win
-    $ cmake -DCMAKE_TOOLCHAIN_FILE=/path to mxe/usr/i686-pc-mingw32/share/cmake/mxe-conf.cmake ..
+    mkdir build-win
+    cd build-win
+    cmake -DCMAKE_TOOLCHAIN_FILE=/path to mxe/usr/i686-pc-mingw32/share/cmake/mxe-conf.cmake ..
 
 Before compiling we have to add the mxe/usr/bin directory to the PATH (so
 windres and makensis can be found):
 
-    $ export PATH=/path to mxe/usr/bin:$PATH
+    export PATH=/path to mxe/usr/bin:$PATH
 
 Now compile:
 
-    $ make
+    make
 
 If you additionally want an NSIS installer:
 
-    $ make package
+    make package
 
-done.
+Done.
 
 ## Build with SQLCipher support
 
-When built with SQLCipher support, DB Browser for SQLite will allow you to open
+When built with SQLCipher support,<br>DB Browser for SQLite will allow you to open
```

Comment:
This seems like a strange place to add a line break?

Reply:
The prompt symbol has been removed because it requires an unnecessary "string editing action" if someone copies and pastes this article. However, if it's important to distinguish between a normal-user and the root account,
we should probably add it back. :)

Reply:
I actually wondering this (it's really personal thing), but in a FHD environment,
I decided that a line break here would be better for readability. 🤔

Reply:
To me, it still looks weird when I try it out on an old 1980x1080p monitor:

![image](https://github.com/sqlitebrowser/sqlitebrowser/assets/406299/049070b2-fa76-439a-a17d-7589ab6f2ea1)

Does it look different for you?

Reply:
Ah, sorry, I was too hasty.
I only considered the screen resoultion, not the font settings, which is a bit different than what I was going for.

Reply:
Resolve via [2e35434](https://github.com/sqlitebrowser/sqlitebrowser/pull/3485/commits/2e35434ed29d7e80e365f70c91b32c45afb58a84)